hazelcast/src/main/java/com/hazelcast/config/NetworkConfig.java

```diff
@@ -409,6 +409,11 @@ public AltoSocketConfig getAltoSocketConfig() {
         return altoSocketConfig;
     }
 
+    public NetworkConfig setAltoSocketConfig(AltoSocketConfig altoSocketConfig) {
+        this.altoSocketConfig = altoSocketConfig;
```

Comment:
Should not be a null check? Or a null config allowed?

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/edefb3f793637b30e23836acbd07b60013525cd3

Reply:
Normally we use Preconditions to check.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/16aaf911c270da78592443f9d81c145858b9459e